### ISSUE TITLE
No BUSCO score of BRAKER3 prediction result with protein only 

### ISSUE THREAD

**Issue by phhsieh001** [Gaius-Augustus/BRAKER#771]
Dear BRAKER Team,

We ran BRAKER(ver. 3.0.7 docker) with rna-seq only and protein only separately at our termite genome. Here are the BUSCO score at inputs and outputs. 
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/phhsieh/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/phhsieh/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

  | BUSCO(insecta_odb10)
-- | --
Genome | C:99.6%[S:97.3%,D:2.3%]
Result of RNA-seq only annotated | C:94.9%[S:74.0%,D:20.9%]
Result of protein only annotated | C:0.0%[S:0.0%,D:0.0%]
the protein evidenve used in BRAKER | C:99.9%[S:15.4%,D:84.5%]

</body>

</html>

What are the possible reasons for causing the zero BUSCO in protein only annotation? Thank you!

**Comment by KatharinaHoff** [comment]
This looks like something went wrong.

Are you sure the file that you applied BUSCO to actually contained any protein sequences at all? (I am currently assuming that you are running BUSCO outside of BRAKER, not using the internal built-in compleasm.)

**Comment by phhsieh001** [comment]
Yes, we ran BUSCO outside of BRAKER. We used docker to run BUSCO in transcript mode with BRAKER result (braker.codingseq) as input. There are 34,060 sequences in braker.codingseq file and the BUSCO: 0% (Ran BRAKER with only protein evidence). There are  71,027 sequences in braker.codingseq file and the BUSCO: 94.9% (Ran BRAKER with only RNA-seq evidence). These results are both appeard in BRAKER2 and BRAKER3.

**Comment by KatharinaHoff** [comment]
[BRAKER4](https://github.com/Gaius-Augustus/BRAKER4) has built-in quality control: BUSCO, compleasm, OMArk, and optional gffcompare all run as separate Snakemake rules in their own containers. Path and version issues are eliminated.
